esp/src/src-react/components/InfoGrid.tsx

```diff
@@ -84,12 +75,12 @@ export const InfoGrid: React.FunctionComponent<InfoGridProps> = ({
                 sortable: false,
                 formatter: React.useCallback(function (Message, idx) {
                     const info = extractGraphInfo(Message);
-                    if (info.graphID && info.subgraphID && info.activityID) {
-                        const txt = "Graph " + info.graphID + "[" + info.subgraphID + "], " + info.activityName + "[" + info.activityID + "]";
-                        return <Link href={`#/workunits/${wuid}/metrics/sg${info.subgraphID}`}>{txt}</Link>;
-                    } else if (info.graphID && info.subgraphID) {
-                        const txt = "Graph " + info.graphID + "[" + info.subgraphID + "]";
-                        return <Link href={`#/workunits/${wuid}/metrics/sg${info.subgraphID}`}>{txt}</Link>;
+                    if (info.graphID && info.subgraphID) {
+                        let txt = `Graph ${info.graphID}[${info.subgraphID}]`;
+                        if (info.activityName && info.activityID) {
+                            txt = `Graph ${info.graphID}[${info.subgraphID}], ${info.activityName} [${info.activityID}]`;
+                        }
+                        return <><Link href={`#/workunits/${wuid}/metrics/sg${info.subgraphID}`}>{txt}</Link>{info.message && <span style={{ marginLeft: 5 }}>{info.message}</span>}</>;
```

Comment:
Looks like your moving the link to the front of the message?

I would exptect something more like:
Before:
**There is an issue with XXXxxx and also YYYyyy, so there...**
After:
**There is an issue with [XXXxxx](https://www.google.com) and also [YYYyyy](https://www.google.com), so there...**